# HG changeset patch
# User Kai Habel <kai.habel@gmx.de>
# Date 1310844408 -7200
#      Sat Jul 16 21:26:48 2011 +0200
# Node ID e3e6079039c64850ef880e06078f6588b275b59a
# Parent  cd59f52719418ea79000f06fd34255127ca1a73f
codesprint: 2 demos for pcolor.m dummy for surface.m

diff --git a/scripts/plot/pcolor.m b/scripts/plot/pcolor.m
--- a/scripts/plot/pcolor.m
+++ b/scripts/plot/pcolor.m
@@ -75,8 +75,19 @@ function h = pcolor (x, y, c)
     set (ax, "view", [0, 90]);
   endif
 
   if (nargout > 0)
     h = tmp;
   endif
 
 endfunction
+
+%!demo
+%! clf
+%! [~,~,Z]=peaks;
+%! pcolor(Z);
+
+%!demo
+%! [X,Y,Z]=sombrero;
+%! [Fx,Fy] = gradient(Z);
+%! pcolor(X,Y,Fx+Fy);
+%! shading interp;
diff --git a/scripts/plot/surface.m b/scripts/plot/surface.m
--- a/scripts/plot/surface.m
+++ b/scripts/plot/surface.m
@@ -150,8 +150,10 @@ function [h, bad_usage] = __surface__ (a
                         other_args{:});
 
     if (! ishold ())
       set (ax, "view", [0, 90], "box", "off");
     endif
   endif
 
 endfunction
+
+%!assert (1)
\ No newline at end of file
